index.test.js

```diff
@@ -99,5 +107,10 @@ describe('babel-preset-zapier', () => {
 
       expect(transform(code)).toMatchSnapshot();
     });
+
+    it('produce the appropriate configuration', () => {
+      const config = tetstConfig();
+      expect(config).toMatchSnapshot();
```

Comment:
Would it be better to assert that `babel-plugin-require-context-hook` is part of the list of plugins here? Otherwise it's kind of vague what this test is asserting on.

Reply:
I had the same approach in https://github.com/zapier/babel-preset-zapier/pull/13.

I think it's good to have a clear snapshot of the config as a whole, a little like we do for components. The main reason is for maintenance, as in the future we may add/remove entries to the config for this environment, it makes the test a little bit more maintainable as we don't have to manually mirror those modifications in the test too.

When #13 gets merged, I also like the fact we can see each different variants of the output config in the snapshot file, to have a clear vision of what it looks like.